producers/docker_trivy/main.go

```diff
@@ -1,77 +1,106 @@
+            log.Fatal(err)
+        }
 
-		for _, vuln := range res.Vulnerable {
-			issues = append(issues, parseResult(&vuln, target))
-		}
-	}
-	return issues
+    }
 }
+func parseCombinedOut(results types.CombinedOut) []*v1.Issue {
+    issues := []*v1.Issue{}
+    for img, output := range results{
+        log.Printf("Parsing Combined Output for %s\n",img)
+        for _, res := range output {
+            target := res.Target
+            for _, vuln := range res.Vulnerable {
+                issues = append(issues, parseResult(&vuln, target))
+            }
+        }
+    }
+        return issues
+    }
+    func parseSingleOut(results []types.TrivyOut) []*v1.Issue {
+        issues := []*v1.Issue{}
+        for _, res := range results {
+            target := res.Target
 
-// TrivySeverityToDracon maps Trivy Severity Strings to dracon struct
-func TrivySeverityToDracon(severity string) v1.Severity {
-	switch severity {
-	case "LOW":
-		return v1.Severity_SEVERITY_LOW
-	case "MEDIUM":
-		return v1.Severity_SEVERITY_MEDIUM
-	case "HIGH":
-		return v1.Severity_SEVERITY_HIGH
-	case "CRITICAL":
-		return v1.Severity_SEVERITY_CRITICAL
-	default:
-		return v1.Severity_SEVERITY_INFO
-	}
-}
+            for _, vuln := range res.Vulnerable {
+                issues = append(issues, parseResult(&vuln, target))
+            }
+        }
+        return issues
+    }
 
-func parseResult(r *types.TrivyVulnerability, target string) *v1.Issue {
-	cvss := r.CVSS.Nvd.V3Score
-	return &v1.Issue{
-		Target:     target,
-		Type:       "Container image vulnerability",
-		Title:      fmt.Sprintf("[%s][%s] %s", target, r.CVE, r.Title),
-		Severity:   TrivySeverityToDracon(r.Severity),
-		Confidence: v1.Confidence_CONFIDENCE_MEDIUM,
-		Cvss:       cvss,
-		Description: fmt.Sprintf("CVSS Score: %v\nCvssVector: %s\nCve: %s\nCwe: %s\nReference: %s\nOriginal Description:%s\n",
-			r.CVSS.Nvd.V3Score, r.CVSS.Nvd.V3Vector, r.CVE, strings.Join(r.CweIDs[:], ","), r.PrimaryURL, r.Description),
-	}
-}
+    // TrivySeverityToDracon maps Trivy Severity Strings to dracon struct
+    func TrivySeverityToDracon(severity string) v1.Severity {
+        switch severity {
+        case "LOW":
+            return v1.Severity_SEVERITY_LOW
+        case "MEDIUM":
+            return v1.Severity_SEVERITY_MEDIUM
+        case "HIGH":
+            return v1.Severity_SEVERITY_HIGH
+        case "CRITICAL":
+            return v1.Severity_SEVERITY_CRITICAL
+        default:
+            return v1.Severity_SEVERITY_INFO
+        }
+    }
+
+    func parseResult(r *types.TrivyVulnerability, target string) *v1.Issue {
+        cvss := r.CVSS.Nvd.V3Score
+        return &v1.Issue{
+            Target:     target,
+            Type:       "Container image vulnerability",
+            Title:      fmt.Sprintf("[%s][%s] %s", target, r.CVE, r.Title),
+            Severity:   TrivySeverityToDracon(r.Severity),
+            Confidence: v1.Confidence_CONFIDENCE_MEDIUM,
+            Cvss:       cvss,
```

Comment:
```suggestion
            Cvss:       r.CVSS.Nvd.V3Score,
```

This eliminates the need for `cvss` to be declared